Formula/mgis.rb

```diff
@@ -0,0 +1,37 @@
+class Mgis < Formula
+  desc "Provide tools to handle MFront generic interface behaviours"
+  homepage "https://thelfer.github.io/mgis/web/index.html"
+  url "https://github.com/thelfer/MFrontGenericInterfaceSupport/archive/refs/tags/MFrontGenericInterfaceSupport-2.0.tar.gz"
+  sha256 "cb427d77f2c79423e969815b948a8b44da33a4370d1760e8c1e22a569f3585e2"
+  license "GPL-1.0-or-later"
+  head "https://github.com/thelfer/MFrontGenericInterfaceSupport.git", using: :git, branch: "master"
+
+  depends_on "boost-python3" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "python@3.9" => :build
```

Comment:
Does it not work with the latest version

Reply:
I don't think so, `boost-python3` seems to depend on `python@3.9`. 
With a dependency on `python@3.10`, the compilation fails, on my mac, because it can't find some boost-python` libraries.

Reply:
Then it needs a comment to that extend